The API server operator should include the SCTP feature gate

dougbtv opened the pull request:
Updates to latest openshift/api 

Method used to update:

```
# go get with commit @ HEAD to include sctp feature gate
go get github.com/openshift/api@4efd1a50a74e455e53d0f4478657e5c226fe9b24
# pick up generated version from a diff
git diff 
go checkout .
# modify the replace section to include generated version from go get above
vi go.mod
go mod vendor
go mod tidy
```

Happy to change that workflow -- the `replace` section was a curve for me. Unsure if this is the proper workflow.

danwinship commented:
from kube-apiserver logs:

    I1114 21:03:45.634381       1 flags.go:33] FLAG: --feature-gates="LegacyNodeRoleBehavior=false,NodeDisruptionExclusion=true,RotateKubeletServerCertificate=true,SCTPSupport=true,ServiceNodeExclusion=true,SupportPodPidsLimit=true"

unit test failed because it needs to be updated for the new feature gate:

    render_test.go:212: "scenario 1 checks feature gates" reports incorrect config file, error: expected to get exactly 6 feature gates but found 7: expected=[RotateKubeletServerCertificate=true SupportPodPidsLimit=true NodeDisruptionExclusion=true ServiceNodeExclusion=true SCTPSupport=true LegacyNodeRoleBehavior=false] got=[ExperimentalCriticalPodAnnotation=true RotateKubeletServerCertificate=true SupportPodPidsLimit=true TLSSecurityProfile=true NodeDisruptionExclusion=true ServiceNodeExclusion=true LegacyNodeRoleBehavior=false]

dougbtv commented:
Thanks for the eyes on the test results!

Updated `bindata/bootkube/config/bootstrap-config-overrides.yaml` to match the flags, ran `go test -race -json ./pkg/... ./cmd/...` and it exited 0, and also rebased.

sttts commented on go.sum:
```diff
@@ -314,10 +301,6 @@ github.com/stretchr/testify v1.2.2/go.mod h1:a8OnRcib4nhh0OaRAV+Yts87kKdq0PP7pXf
-github.com/sttts/library-go v0.0.0-20191022152013-4b1f730d0879b1eeeb5d3139947e31fc7dea1747 h1:isD2EcnWHnaJF8REoIm2xjMq2te1HK3GIKICKKFdCAg=
-github.com/sttts/library-go v0.0.0-20191022152013-4b1f730d0879b1eeeb5d3139947e31fc7dea1747/go.mod h1:7jxUj8s/i6L5GilDCNZJGuisFUB0c/I6IU7ralc7fNs=
-github.com/sttts/library-go v0.0.0-20191108112529-6cd624a786bd h1:eQ1sF2CfL16PqvZYmf4c5NzitKX5DE9y5LtvU9RKj9g=
-github.com/sttts/library-go v0.0.0-20191108112529-6cd624a786bd/go.mod h1:7jxUj8s/i6L5GilDCNZJGuisFUB0c/I6IU7ralc7fNs=
```
/hold

we have to fix this. My fork.

sttts replied:
Or you did with this PR? Then even better.

sttts commented:
/hold cancel
/lgtm
/approve
